static/images/Makefile

```diff
@@ -1,4 +1,4 @@
-all: sponsor-github.png sponsor-hetzner.png sponsor-hoenscheiditconsulting.png sponsor-betadots.png sponsor-babiel.png sponsor-enterprise-modules.png sponsor-docker.png sponsor-puppet.png sponsor-deubert.png sponsor-microsoft.png
```

Comment:
the `sponsor-hoenscheiditconsulting.png` file got deleted earlier, but it wasn't deleted from the Makefile.